src/components/Review/SectionProgress.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Icon, Label, Progress, Segment } from 'semantic-ui-react'
+import { ReviewProgress, SectionProgress, SectionStateNEW } from '../../utils/types'
+import strings from '../../utils/constants'
+
+const SectionProgress: React.FC<SectionStateNEW> = ({ assigned, reviewProgress }) => {
+  const getProgressTitle = ({ doneNonConform, doneConform, totalReviewable }: ReviewProgress) => {
+    if (doneNonConform > 0)
+      return strings.LABEL_REVIEW_DECLINED.replace('%1', doneNonConform.toString())
```

Comment:
(not required change)
I wonder if template literal would be better here (so that when string are localised, localiser doesn't get confused, or accidentally deletes %1)
i.e `${doneNonConform} ${strings.LABEL_REVIEW_DECLINED}`

Reply:
Yes. I agree. There is another one, should I go change both?

Reply:
Never mind, there isn't another one. Done: f493e1c